### PR TITLE
Ucified version of Zabbix3 packages with SSL suport

### PR DESCRIPTION
**Pull request by limosek**
This is ucified version of Zabbix3 packages. To avoid conflicts with old packages, name of packages are zabbix3-agentd, zabbix3-proxy, zabbix3-get, zabbix3-sender and zabbix3-server.
It supports SSL.

### CODE REVIEW

File: admin/zabbix3/Makefile [hunk 1]
```diff
@@ -8,12 +8,21 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=zabbix3
-PKG_VERSION:=3.0.0beta1
+<<<<<<< HEAD
```
Comment (hnyman) [inline admin/zabbix3/Makefile]: There are merge conflict leftovers in the last commit. At least from line 10 and line 50 onward...

And you need to sign-off-by your commits.

**Comment by hnyman** [comment]
CC also to the zabbix maintainer:  @champtar 

And I don't believe that both zabbix2 and zabbix3 should be present in the sources, unless there is a major need for that.

**Comment by champtar** [comment]
hi @limosek,
you have 3 different topics here: zabbix update to 3.0, a new zabbix-agentd-ssl, and a new zabbix-uci package.
I'll try to work on first 2 soon, and we can discuss uci integration after.
I've seen your mail, in the future just write in the PR so everyone can see the discussion.

**Comment by champtar** [comment]
i've pushed zabbix 3.0.1
https://github.com/openwrt/packages/commit/fe88e0447ab21e73d347f0933273d2859493ba9d

**Comment by hnyman** [comment]
zabbix has already been updated to 3.0.3 since this PR was filed.

**Pull request closed without merging**